make/autoconf/flags-cflags.m4

```diff
@@ -133,7 +133,7 @@ AC_DEFUN([FLAGS_SETUP_WARNINGS],
       BUILD_CC_DISABLE_WARNING_PREFIX="-wd"
       CFLAGS_WARNINGS_ARE_ERRORS="-WX"
 
-      WARNINGS_ENABLE_ALL="-W3"
+      WARNINGS_ENABLE_ALL="-W3 -Zc:strictStrings"
```

Comment:
This is not strictly a flag to enable warnings. I'd recommend you move it to line 499 and 500 in the same file, which would align it with the -Zc:wchar_t- switch. Make sure you add it to both TOOLCHAIN_CFLAGS_JVM and TOOLCHAIN_CFLAGS_JDK.

Reply:
Done

Reply:
Did you forget to push the fix?

Reply:
Push works better when connected... this time it's pushed for real.